.github/workflows/ci.yaml

```diff
@@ -37,6 +37,7 @@ jobs:
       uses: ruby/setup-ruby@v1
       with:
         ruby-version: "${{ matrix.ruby-version }}"
+        bundler: "${{ matrix.ruby-version == '3.1' && 'latest' || 'default' }}"
```

Comment:
Alternatively, I could just set it to `latest` for everything, but I worry that we won't follow up with a removal when we drop 3.1.  This way we can't miss it.  Thoughts?